### ISSUE TITLE
HTML: add `prefers-color-scheme: dark` to the default css

### ISSUE THREAD

**Issue by l0r3m1psum** [jgm/pandoc#7125]
Hello I would like to add `prefers-color-scheme: dark` media query to the default CSS generated when using this command `pandoc -s -o f.html f.md`, something like this:

```
@media (prefers-color-scheme: dark) {
  html {
    background-color: #1E1F21;
    color: #EEEFF1;
  }
  a {
    color: #EEEFF1;
  }
  a:visited {
    color: #EEEFF1;
  }
  blockquote p: {
    color: #606060;
  }
  hr {
    background-color: #EEEFF1;
  }
}
```

My rationale behind this enhancement is to make it easier to read the rendered HTML at night by default.

**Comment by mb21** [comment]
Yes, I did something similar [here](https://github.com/mb21/panwriter/blob/master/static/preview.pandoc-styles.css#L135).

Works great until the user starts setting the `fontcolor`, `linkcolor` or `backgroundcolor` [variables](https://pandoc.org/MANUAL.html#variables-for-html). We could also add `fontcolordark`, `linkcolordark` and `backgroundcolordark` of course...

**Comment by l0r3m1psum** [comment]
Ok what you did [there](https://github.com/mb21/panwriter/blob/master/static/preview.pandoc-styles.css#L135) looks better.

> Works great until the user starts setting the fontcolor, linkcolor or backgroundcolor variables. We could also add fontcolor-dark, linkcolor-dark and backgroundcolor-dark of course...

Or you could accept a couple of colors, where the first is used for the light theme and the second for the dark one, if just one is given the same color is used for both themes.

**Comment by mb21** [comment]
> Or you could accept a couple of colors,

You mean as a list or array?

I just checked again, and actually when we just overwrite the CSS values, the dark mode will be unchanged when the user sets custom variables. Maybe that's actually for the best, as perhaps the user wouldn't test dark mode anyway when setting variables? I've made a pull request: https://github.com/jgm/pandoc/pull/7131

**Comment by l0r3m1psum** [comment]
> You mean as a list or array?

 I meant a YAML list, assuming we are talking about a [YAML metadata block](https://pandoc.org/MANUAL.html#extension-yaml_metadata_block).

> I've made a pull request: #7131

Let's hope it gets merged :).
